lib/numo/linalg/autoloader.rb

```diff
@@ -87,17 +96,36 @@ def find_libs(lib_names, lib_dirs)
         lib_ext = detect_library_extension
         lib_arr = lib_names.map do |l|
           x = nil
-          lib_dirs.find do |d|
-            x = Dir.glob("#{d}/lib#{l}{,64}.#{lib_ext}{,.*}").find do |lib|
+          if windows?
+            # On Windows, try to search the default DLL search path at first
+            [
+              "lib#{l}.#{lib_ext}",
+              "lib#{l}64#{lib_ext}"
```

Comment:
If `.` is placed after `64`, I think the library detection will be more certainly:
```ruby
"lib#{l}64.#{lib_ext}"
```

Reply:
Oops!  I've fixed it.  Thank you for catching this.